### PR TITLE
Merge of tables-headers-id rules

### PR DESCRIPTION
**Pull request by kensgists**
Merging the two tables-headers-id rules back into a single semi-auto rule.

### CODE REVIEW

File: _drafts/SC1-3-1-tables-headers-id.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+### step3-pass1
+
+| Property | Value
+|----------|----------
+| type     | TestResult
+| outcome  | Passed
+
+## Implementation Tests
+
+Implementation tests are available at: <placeholder>
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id.md]: This needs tests.

File: _drafts/SC1-3-1-tables-headers-id.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+For each value in each cell `headers` attribute,
+
+find the `th` referenced,
+
+give the user the following question:
+
+| Property             | Value
+|----------------------|---------
+| Presented item       | Highlight a table cell and the related header cell. Display the sequence order of the header cell as a number value, as specified by the order in which the header's `id` appears within the cell's `headers` attribute.
+| Question             | Is the number of the related header consistent with the desired reading sequence?
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id.md]: Open question: Is this a WCAG violation or best practice? Please add documentation to the background as evidence.

File: _drafts/SC1-3-1-tables-headers-id.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+## Description
+
+This rule checks that data table `headers` attributes have values that reference `th` or `td` elements in the same table and that these headers-id associations reference correct targets in the correct order.
+
+## Background
+
+- [F90: Failure of Success Criterion 1.3.1 for incorrectly associating table headers and content via the headers and id attributes](https://www.w3.org/TR/WCAG20-TECHS/F90.html)
+- [H43: Using id and headers attributes to associate data cells with header cells in data tables](https://www.w3.org/TR/WCAG20-TECHS/H43.html)
+- [F46: Failure of Success Criterion 1.3.1 due to using th elements, caption elements, or non-empty summary attributes in layout tables](https://www.w3.org/TR/WCAG20-TECHS/F46.html)
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id.md]: Comment Charu: This isn't tested in the rule. Should probably be updated.

File: _drafts/SC1-3-1-tables-headers-id.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+## Assumptions
+
+- Tables are [well-formed, according to the HTML5.1 specification.](https://www.w3.org/TR/html51/tabular-data.html#forming-a-table)
+- Tables using headers-id associations are intended as data rather than layout tables.
+
+## Test procedure
+
+### Selector
+
+Select all elements that match `th[headers]` or `td[headers]`.
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id.md]: We should exclude `role=presentation` / `role=none`.

File: _drafts/SC1-3-1-tables-headers-id-correct.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id-correct.md]: Should be removed, in favor of tables-headers-id.

File: _drafts/SC1-3-1-tables-headers-id-valid.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
```
Comment (WilcoFiers) [inline _drafts/SC1-3-1-tables-headers-id-valid.md]: Should be removed, in favor of tables-headers-id.

File: _drafts/SC1-3-1-tables-headers-id.md (hunk 4 above)
Comment (EmmaJP) [reply _drafts/SC1-3-1-tables-headers-id.md]: Yep, any headers stuff incorrectly applied on presentation tables should be picked up by a different rule.

**Pull request merged**